src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100, hidden = false) {
+    this.name = name;
+    this.health = health;
+    this.hidden = hidden;
```

Comment:
The 'hidden' property should not be part of the Animal class constructor as it is specific to Herbivores. It's better to define it directly within the Herbivore class.